db/migrate/20211203100649_add_package_name_to_benchmarks.rb

```diff
@@ -0,0 +1,16 @@
+class AddPackageNameToBenchmarks < ActiveRecord::Migration[5.2]
+  def up
+    add_column :benchmarks, :package_name, :string
+
+    # Update already imported Benchmarks with their package names
+    SupportedSsg.all.each do |ssg|
+      ref_id = "#{Xccdf::Benchmark::REF_PREFIX}-#{ssg.os_major_version}"
+      Xccdf::Benchmark.where(ref_id: ref_id, version: ssg.version)
+                      .update(package_name: ssg.package)
```

Comment:
@skateman how do you feel about trying it out with reimport. Empty packages would cause a reimport which should set the correct package on benchmark objects. This way we should be sure that what we have imported is actually what is coming out of package that is (going to be) set.
Although, there a potential risk of this deadlock, as it it would/should be have as a force import.